site/en/blog/deps-rems-95/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Deprecations and removals in Chrome 95
+description: >
+  A round up of the deprecations and removals in Chrome 95 to help you plan.
+layout: 'layouts/blog-post.njk'
+date: 2021-09-23
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/0GrO9wVaQsRjPJQP9fc0.png'
+alt: >
+  Deprecations and Removals hero logo
+tags:
+  - deprecations
+  - removals
+  - chrome-95
+---
+
+{% include 'partials/see-all-dep-rem.md' %}
+
+Chrome 95 beta was released on September 23, 2021 and is expected to become the
+stable version in late October, 2021.
+
+## FTP Support Removed
```

Comment:
```suggestion
## FTP support removed
```

Reply:
Sorry about that. I copy this from the blog post where they do it the other way.